iai-callgrind-runner/tests/test_tools.rs

```diff
@@ -0,0 +1,44 @@
+use std::path::PathBuf;
+
+use iai_callgrind_runner::runner::costs::Costs;
+use iai_callgrind_runner::runner::dhat::logfile_parser::DhatLogfileParser;
+use iai_callgrind_runner::runner::tool::logfile_parser::{LogfileParser, LogfileSummary};
+fn dummy_summary(cmd: &str, pid: i32, cost_name: &str, cost: u64) -> LogfileSummary {
+    let costs = Costs::with_event_kinds([(cost_name.to_string(), cost)]);
+    LogfileSummary {
+        command: cmd.parse().unwrap(),
+        pid,
+        parent_pid: None,
+        fields: vec![],
+        details: vec![],
+        error_summary: None,
+        costs: Some(costs),
+        log_path: PathBuf::new(),
+    }
+}
+
+#[test]
+fn test_dhat_merge_logfile_summaries() {
```

Comment:
You can use `rstest` if you like to. It's already in the dev-dependencies.

Reply:
If its ok, I think I'll just use `#[test]`